cypress/fixtures/analysis.json

```diff
@@ -64,6 +64,16 @@
     {
         "source": "Upload a local binary",
         "target": ["Jakarta EE 9"],
-        "binary": ["acmeair-webapp-1.0-SNAPSHOT.war", "customers-tomcat-0.0.1-SNAPSHOT.war"]
+        "binary": [
+            "acmeair-webapp-1.0-SNAPSHOT.war",
+            "customers-tomcat-0.0.1-SNAPSHOT.war",
+            "spring-petclinic-2.4.0.BUILD-SNAPSHOT.jar"
+        ],
+        "appName": [
```

Comment:
Can't we use the data from binary itself? 
 I see that only story points are extra.
Can we add story points to binary itself ?

Reply:
Rebase the PR

Reply:
@sshveta rebased the PR.